8_september/ts_assignment/Script/eight.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+var _a;
```

Comment:
remove if unused. Use proper variable name

Reply:
Variable Name Changed In TypeScript file